molecule/kerberos-rhel/verify.yml

```diff
@@ -6,6 +6,9 @@
   hosts: kafka_broker
   gather_facts: false
   tasks:
+    - import_role:
+        name: variables
```

Comment:
if we are importing all the variables just to check kraft_enabled, I think its a better idea to use `groups['kafka_controller'] is defined`

Reply:
done